classification/sampling.csv

```diff
@@ -1,14 +1,22 @@
 Authors,Venue,Year,Title,Algorithm,Input Data,Algorithm Category,Coverage,Evaluation,Application,Further Tags
-"Jeho Oh, Paul Gazzillo, Don Batory, Marijn Heule, Maggie Myers",(techreport),2019,"Uniform Sampling from Kconfig Feature Models","","","","","","",""
-" Daniel-Jesus Munoz,  Jeho Oh,  Monica Pinto,  Lidia Fuentes,  Don Batory",SPLC,2019,"Uniform Random Sampling Product Configurations of Feature Models That Have Numerical Features","","","","","","",""
+"Juliana Alves Pereira, Mathieu Acher, Hugo Martib, Jean-Marc Jezequel",Proceedings of the ACM/Standard Performance Evaluation Corporation International Conference on Performance Engineering,2020,"Sampling Effect on Performance Prediction of Configurable Systems: A Case Study","","","","","effectiveness, available tool, open-source tool, evaluation","non-functional properties","compares Coverage-based/Solver-based/Randomized solver-based/Distance-based/Diversified distance-based/Random"
```

Comment:
Always use abbreviations for conferences

Reply:
Check